doc/users/whats_new.rst

```diff
@@ -409,28 +439,35 @@ to ``False``.
 
 Example
 ~~~~~~~
-::
+.. plot::
+    :include-source:
+    :align: center
 
     import numpy as np
     import matplotlib.pyplot as plt
     from mpl_toolkits.mplot3d import Axes3D
 
-    fig = plt.figure(figsize=(7,3))
-    ax1 = fig.add_subplot(121, projection='3d')
     x = np.arange(2)
     y = np.arange(3)
     x2d, y2d = np.meshgrid(x, y)
-    x2d, y2d = x2d.ravel(), y2d.ravel()
-    z = x2d + y2d
-    ax1.bar3d(x2d, y2d, x2d * 0, 1, 1, z, shade=True)
+    x, y = x2d.ravel(), y2d.ravel()
+    z = np.zeros_like(x)
+    dz = x + y
 
-    ax2 = fig.add_subplot(122, projection='3d')
-    ax2.bar3d(x2d, y2d, x2d * 0, 1, 1, z, shade=False)
-    fig.canvas.draw()
+    fig = plt.figure(figsize=(4, 6))
+    ax1 = fig.add_subplot(2, 1, 1, projection='3d')
+    ax1.bar3d(x, y, z, 1, 1, dz, shade=True)
+    ax1.set_title('Shading On')
 
+    ax2 = fig.add_subplot(2, 1, 2, projection='3d')
+    ax2.bar3d(x, y, z, 1, 1, dz, shade=False)
+    ax2.set_title('Shading Off')
+
+    plt.show()
 
-New which Parameter for autofmt_xdate
--------------------------------------
+
+New which Parameter for ``autofmt_xdate``
+-----------------------------------------
 
 A ``which`` parameter now exists for the method
 :func:`~matplotlib.figure.Figure.autofmt_xdate`. This allows a user to format
```

Comment:
Just below: I would just say the default is "major" (... for all purposes).  It's also not just rotating, but also aligning.